src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileZipGeneratorImpl.java

```diff
@@ -51,16 +78,71 @@ public Path generateAndWriteZip(List<List<AudioFileInfo>> audioSessions) {
         }
     }
 
-    private Map<Path, Path> generateZipStructure(List<List<AudioFileInfo>> audioSessions) {
+    private ViqMetaData createViqMetaData(MediaRequestEntity mediaRequestEntity) {
+        return ViqMetaData.builder()
+            .courthouse(mediaRequestEntity.getHearing().getCourtroom().getCourthouse().getCourthouseName())
+            .raisedBy(null)
```

Comment:
Should the raised by be null?

Reply:
That's my understanding currently when I refer to the readme in the other ticket [DMP-588](https://tools.hmcts.net/jira/browse/DMP-588) ATS - VIQ Header - Readme.txt file

readMe.txt shows `Raised by: `

https://github.com/hmcts/darts-legacy/blob/2e512caf650aeb532fbdad539a714cd191a7144c/workspace/dartsdev/darts-ats/src/main/java/com/cgi/darts/ats/MojDownloadRequest.java#L259C27-L259C36 shows that it appears to only being populated for Transcription Requests.

It seems to also be missing the empty lines between the fields - just an observation